lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -338,6 +340,8 @@ def _section_data_download(course, access):
 def _section_send_email(course, access):
     """ Provide data for the corresponding bulk email section """
     course_key = course.id
+    queries = GroupedQuery.objects.filter(course_id=course_key)
+    query_options = tuple((query.id, query.title if query.title else u'Query saved at ' + query.created.strftime("%m-%d-%y %H:%M")) for query in queries)
```

Comment:
This is a bit long and hard to grok; could we split it out onto multiple lines?

```python
query_options = tuple(
    (query.id, query.title if query.title else u'Query saved at ' + query.created.strftime("%m-%d-%y %H:%M"))
    for query in queries
)
```

Also, could we simplify the inline conditional like this?
```python
(query.id, query.title or u'Query saved at ' + query.created.strftime("%m-%d-%y %H:%M"))
```